iphone/Maps/Classes/Components/MWMTableViewController.m

```diff
@@ -19,6 +19,23 @@ - (BOOL)prefersStatusBarHidden
   return NO;
 }
 
+- (void)viewWillAppear:(BOOL)animated
+{
+  [super viewWillAppear:animated];
+  
+  for (NSInteger i = 0; i < [self numberOfSectionsInTableView:self.tableView]; i++) {
+    [self updateHeaderView:[self.tableView headerViewForSection:i]];
+  }
+}
+
+// Fix table section header font color for all tables, including Setting and Route Options.
+- (void)updateHeaderView:(UIView*)headerView {
+  if ([headerView isKindOfClass: [UITableViewHeaderFooterView class]]) {
+    UITableViewHeaderFooterView* header = (UITableViewHeaderFooterView *)headerView;
+    header.textLabel.textColor = [UIColor blackSecondaryText];
```

Comment:
There is also detailTextLabel, does it influence the (not used) bottom footer? Does it make sense to fix it here too (and quick-test if it works, by adding some test text and then removing it)?

Reply:
Let’s do that, what color it should have in your opinion? The same as textLabel?

Reply:
See https://developer.apple.com/documentation/uikit/uitableviewheaderfooterview/1624910-detailtextlabel?language=objc

> This property is only for tables configured with [UITableViewStyleGrouped](https://developer.apple.com/documentation/uikit/uitableviewstyle/uitableviewstylegrouped?language=objc).

It seems we use `[UITableViewStylePlain](https://developer.apple.com/documentation/uikit/uitableviewstyle/uitableviewstyleplain?language=objc)` which is default and it is quite difficult to check how it will work with `UITableViewStyleGrouped`.

So I would propose to update `detailTextLabel.textColor` to not have this problem in the future if by any chance we will ever use `UITableViewStyleGrouped` and that's it(or even just do nothing). WDYT?

Reply:
1. Bottom (footer) texts under sections are available only in grouped table style, right?
2. Can we use `if` and modify it only if it's a grouped style? And ideally also only if it's present (grouped style may not have a footer, it's optional, right?).

Reply:
> Bottom (footer) texts under sections are available only in grouped table style, right?

It seems it can be in header of section too, not only footer(it is just additional small text near the main one), but right. 

> Can we use if and modify it only if it's a grouped style? And ideally also only if it's present (grouped style may not have a footer, it's optional, right?).

I will try.

Reply:
@biodranik just updated, PTAL. I didn't make any changes for the "And ideally also only if it's present" part, because I am not sure how can we properly check if it is the case(after all I doubt we are going to use grouped table views and we will we always can add some slight fixes afterwards)